fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -600,6 +604,13 @@ protected Ratio getRatio() {
                     }
             );
         }
+
+        if (!metrics.containsKey(UPTIME)) {
+            MetricRegistryFactory.getRegistry().register(
+                    UPTIME,
+                    (Gauge<Long>) () -> ManagementFactory.getRuntimeMXBean().getUptime()
```

Comment:
`metricRegistry` is already defined as `ManagementFactory.getRuntimeMXBean()`, so you may use that.